FakeNFT/Scenes /Catalogue/CollectionScene/CollectionViewController.swift

```diff
@@ -0,0 +1,279 @@
+		let label = UILabel()
+		label.font = .bold22
+		return label
+	}()
+	
+	private lazy var authorLink: UILabel = {
+		let label = UILabel()
+		label.font = .caption1
+		label.textColor = .blue
+		label.isUserInteractionEnabled = true
+		let tapGesture = UITapGestureRecognizer(target: self, action: #selector(didTapAuthorLink))
+		label.addGestureRecognizer(tapGesture)
+		return label
+	}()
+	
+	private lazy var authorTitle: UILabel = {
+		let label = UILabel()
+		label.font = .caption2
+		label.text = "Автор коллекции:"
+		return label
+	}()
+	
+	private let authorHorizontalStackView: UIStackView = {
+		let stackView = UIStackView()
+		stackView.axis = .horizontal
+		stackView.alignment = .firstBaseline
+		stackView.spacing = 4
+		return stackView
+	}()
+	
+	private lazy var nftDescriptionLabel: UILabel = {
+		let label = UILabel()
+		label.font = .caption2
+		label.numberOfLines = 0
+		return label
+	}()
+	
+	private let fullDescriptionVerticalStackView: UIStackView = {
+		let stackView = UIStackView()
+		stackView.axis = .vertical
+		stackView.alignment = .leading
+		stackView.spacing = 5
+		return stackView
+	}()
+
+	private lazy var nftCollectionView: UICollectionView = {
+		let collection = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+		collection.backgroundColor = .clear
+		collection.dataSource = self
+		collection.delegate = self
+		collection.allowsMultipleSelection = false
+		collection.register(CollectionNFTCell.self)
+		return collection
+	}()
+	
+	private lazy var contentView: UIView = {
+		let contentView = UIView()
+		return contentView
+	}()
+
+	private var nftCollectionModel: Collection?
+	private var viewModel: CollectionViewModel?
+	
+	private let collectionConfig = UICollectionView.Config(
+		cellCount: 3,
+		leftInset: 16,
+		rightInset: 16,
+		topInset: 0,
+		bottomInset: 0,
+		height: 192,
+		cellSpacing: 8
+	)
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		setupView()
+		bindViewModel()
+		setupValuesForUIElements()
+		
+		guard let collection = nftCollectionModel else { return }
+		viewModel?.loadNFTForCollection(collection: collection)
+		viewModel?.getAuthorURL(collection: collection)
+	}
+	
+	func initialise(viewModel: CollectionViewModel, model: Collection) {
+		self.viewModel = viewModel
+		self.nftCollectionModel = model
+	}
+
+	@objc
+	private func didTapBackButton() {
+		navigationController?.popViewController(animated: true)
+	}
+	
+	@objc
+	private func didTapAuthorLink() {
+		let aboutAuthorScreen = AboutAuthorViewController()
+		aboutAuthorScreen.modalPresentationStyle = .fullScreen
+		aboutAuthorScreen.inialise(authorPageURL: viewModel?.authorModel.website ?? "")
+		
+		navigationController?.pushViewController(aboutAuthorScreen, animated: true)
+	}
+
+	private func bindViewModel() {
+		guard let viewModel else { return }
+
+		viewModel.$nftItems.bind { [weak self] _ in
+			guard let self = self else { return }
+			self.nftCollectionView.reloadData()
```

Comment:
тут в одну строку реализация, достаточно вызвать  self?
```suggestion
			self?.nftCollectionView.reloadData()
```

Reply:
Это уже привычка, так научили. Возьму на вооружение